### ISSUE TITLE
Kill Aura fine tuning

### ISSUE THREAD

**Issue by VKartze** [THEREALWWEFAN231/wwe#27]
**Minecraft Version**
1.14.2 / any
**What is the suggestion**
My suggestion is to add a customizable filter, like _.killaura add mobname_
**Why it will be useful**
Currently killaura with "mobs" feature enabled affects neutrals(_village_ golems), friendly(squids, villagers) and hostile mobs, not only hostiles. This makes difficult e.g. to protect village from raiders, cuz you're killing not only raiders but also golems and villagers, and this is no good. 
With a filter provided, u can have killaura enabled all the time without worrying about hurting mobs u don't need to hurt.

**Comment by THEREALWWEFAN231** [comment]
Was going add a gui for this at one point, but it was just horrid, then i forgot about this, so thats a good idea, ill checkout it hitting neutral mobs(i made it do this but didn't know it would be a big deal)

**Comment by Paralusion** [comment]
The whole killaura needs a revamp anyway. It may be a good thing to just recreate the whole thing.

**Comment by THEREALWWEFAN231** [comment]
> The whole killaura needs a revamp anyway. It may be a good thing to just recreate the whole thing.

I dont think it needs revamped... it works fine, and has alot of settings, yeah most people dont know what they do, but they are there, also the targeting isn't "killaura" it's doing that for a couple of modules that use the same method

**Comment by Paralusion** [comment]
merged with: #32: add entites to search like: .search add creeper

**Comment by Paralusion** [comment]
merge with: #603  Being able to select specific entities with tracers and ESP, kinda like meteor

**Comment by Paralusion** [comment]
merge with:  #519: Make Tracers customizable, like with .tracer add (mob/animal/username) and .tracer remove (mob/animal/username)
